packages/react-native/native-js-interface.md

```diff
@@ -33,85 +149,105 @@ The `breadcrumb` map/dictionary must have the following structure in order to be
 | `metaData` | Map<String, String> | optional  |
 
 
-<sup>1</sup> This cannot be constrained by the type signature because these arguments go across the React Native bridge, and so only a predetermined set of crude data types can be specified.
+#### `BugsnagReactNative.startSession(): void`
 
-### `BugsnagReactNative.updateClientState(updates): void`
+Calls straight through to the native `startSession()` method.
 
-Updates to mutable client state in JS (e.g. `metaData`, `user`) should be replicated and stored in the native layer.
+#### `BugsnagReactNative.stopSession(): void`
 
-- Android: `void updateClientState(ReadableMap updates)`
-- iOS `- (void)updateClientState:(NSDictionary *)updates;`
+Calls straight through to the native `stopSession()` method.
 
-The keys in the updates map are the names of the properties to be updated and the values are encoded objects containing the new value for that property. The list of allowed keys is:
+#### `BugsnagReactNative.resumeSession(): Promise<boolean>`
```

Comment:
This one's gotta be async if we want a return value, but thankfully we didn't create this method in JS yet.